_includes/_inventory/resources.md

```diff
@@ -7,23 +7,23 @@ interested in isn't listed, please open an issue or contribute!
+| [App Engine Apps](https://cloud.google.com/appengine/docs/admin-api/reference/rest/v1/apps) | Done | TODO | TODO |
+| [BigQuery](https://cloud.google.com/bigquery/docs/reference/rest/v2/) | Done | Done | TODO |
 | [Buckets](https://cloud.google.com/storage/docs/json_api/v1/buckets#resource) | Done | Done | TODO |
 | [BucketAccessControls](https://cloud.google.com/storage/docs/json_api/v1/bucketAccessControls#resource) | Done | Done | TODO |
 | [Cloud SQL](https://cloud.google.com/sql/docs/mysql/admin-api/v1beta4/instances#resource) | Done | Done | TODO |
-| [Compute Engine Backend Services](https://cloud.google.com/compute/docs/reference/latest/backendServices#resource) | Done | In Progress | TODO |
-| [Compute Engine Instances](https://cloud.google.com/compute/docs/reference/latest/instances#resource) | Done | In Progress | TODO |
-| [Compute Engine Instance Groups](https://cloud.google.com/compute/docs/reference/latest/instanceGroups#resource) | Done | In Progress | TODO |
-| [Compute Engine Instance Group Managers](https://cloud.google.com/compute/docs/reference/latest/instanceGroupManagers) | Done | In Progress | TODO |
-| [Compute Engine Instance Templates](https://cloud.google.com/compute/docs/reference/latest/instanceTemplates) | Done | In Progress | TODO |
+| [Compute Engine Backend Services](https://cloud.google.com/compute/docs/reference/latest/backendServices#resource) | Done | Done | TODO |
+| [Compute Engine Instances](https://cloud.google.com/compute/docs/reference/latest/instances#resource) | Done | Done | TODO |
+| [Compute Engine Instance Groups](https://cloud.google.com/compute/docs/reference/latest/instanceGroups#resource) | Done | Done | TODO |
+| [Compute Engine Instance Group Managers](https://cloud.google.com/compute/docs/reference/latest/instanceGroupManagers) | Done | Done | TODO |
+| [Compute Engine Instance Templates](https://cloud.google.com/compute/docs/reference/latest/instanceTemplates) | Done | Done | TODO |
 | [Firewalls](https://cloud.google.com/compute/docs/reference/latest/firewalls) | Done | TODO | Done |
 | [Folders](https://cloud.google.com/resource-manager/reference/rest/v2beta1/folders) | Done | Done | TODO |
 | [Folders Cloud IAM Policies](https://cloud.google.com/iam/reference/rest/v1/Policy) | Done | Done | TODO |
 | [GSuite Groups](https://developers.google.com/admin-sdk/directory/v1/guides/manage-groups) | Done | Done | TODO |
 | [GSuite Group Members](https://developers.google.com/admin-sdk/directory/v1/guides/manage-group-members) | Done | Done | TODO |
-| [IAM GrantableRoles](https://cloud.google.com/iam/reference/rest/v1/roles/queryGrantableRoles) | In Progress | TODO | TODO |
-| [Load Balancer Forwarding Rules](https://cloud.google.com/compute/docs/reference/latest/forwardingRules#resource) | Done | In Progress | TODO |
+| [IAM GrantableRoles](https://cloud.google.com/iam/reference/rest/v1/roles/queryGrantableRoles) | Done | TODO | TODO |
```

Comment:
Sorry for my fault for not being clear, this is inventory and is still in progress.